index.html

```diff
@@ -436,6 +444,11 @@ <h2>
           {{Document}} |document:Document| are as follows:
         </p>
         <ol class="algorithm">
+          <li>If |document:Document|.{{Document/[[OverridePosture]]}} contains
+          "{{DevicePostureType/continuous}}" or "{{DevicePostureType/folded}}",
```

Comment:
Instead of checking for specific values here, I'd just say the equivalent of "if it's not null, return it" (i.e. instead of checking for the presence of specific values, check for the absence of value).